client/src/controllers/SlugController.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller } from '@hotwired/stimulus';
+import { cleanForSlug } from '../utils/text';
+
+/**
+ * <input
+ * type="text"
+ * name="slug"
+ * value="slug testing page"
+ * maxlength="255"
+ * aria-describedby="panel-child-promote-child-for_search_engines-child-slug-helptext"
+ * required=""
+ * id="id_slug">
+ */
+export class SlugController extends Controller<HTMLInputElement> {
+  static values = {
+    allowUnicodeSlugs: { default: true, type: Boolean },
+  };
+
+  declare allowUnicodeSlugsValue: boolean;
+
+  slugify() {
+    this.element.value = cleanForSlug(this.element.value, false, {
+      unicodeSlugsEnabled: true,
```

Comment:
```suggestion
      unicodeSlugsEnabled: this.allowUnicodeValue
```
We should be using the stimulus value here.